cabot/cabotapp/migrations/0026_auto__add_field_instance_hipchat_room_id__add_field_service_hipchat_ro.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Instance.hipchat_room_id'
+        db.add_column(u'cabotapp_instance', 'hipchat_room_id',
```

Comment:
Why do we need a column in cabotapp_instance? We just need it in service, right?